docs/node/nodes/opendss.md

```diff
@@ -0,0 +1,114 @@
+---
+hide_table_of_contents: true
+---
+
+# OpenDSS
+
+OpenDSS is an electric power distribution system simulator (DSS) designed to support distributed energy resource (DER) grid integration and grid modernization.
+This node type run OpenDSS within VILLASnode and enabled VILLASnode to send and receive data to OpenDSS. 
+
+## Prerequisites
+This node type requires OpenDSS shared library. 
+The documentation for compiling the library can be found on [OpenDSS documentation website](https://opendss.epri.com/OpenDSSC.html).
+
+## Implementation
+
+The Implementation of this node type is limited to only sending the data to set the values of certain element including:
+
+ - **Load:** Voltage(kV), Active Power(kW), Apparent Power(kVA), Power Factor(Pf)
+ - **Generator:** Voltage(kV), Active Power(kW), Apparent Power(kVA), Power Factor(Pf)
+ - **ISource:** Current(Amps), Phase angle(AngleDeg), Frequency(f)
+
+For the output, VILLASnode will read the data from monitor element for each time step.
+The data input and output data needed to be transported by [`path`](../config/paths.md) and additional node.
+
+
+The source code of the node-type is available here:
+https://github.com/VILLASframework/node/blob/master/lib/nodes/opendss.cpp
+
+## Configuration {#config}
+
+For input data the name, element type and type of data are needed to be provived.
+However, for output data only the name of the monitor element is needed.
+
+import ApiSchema from '@theme/ApiSchema';
+
+<ApiSchema id="node" example pointer="#/components/schemas/opendss" />
+
+## Example
+
+``` url="external/node/etc/examples/nodes/opendss.conf" title="node/etc/examples/nodes/opendss.conf"
+node = {
+    opendss_node = {
+        type = "opendss"
+
+        #Path to OpenDSS file
+        file_path = "OpenDSS_file/sample.DSS"
```

Comment:
Can we incorporate the OpenDSS example model into the VILLASnode repo and add a link here?